packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -974,6 +979,33 @@ export class StripeHelper {
     return subscriptions;
   }
 
+  /**
+   * Find all active subscriptions for the given `planId`.
+   *
+   * It is expected that the `customer` is expanded.
+   */
+  async *findActiveSubscriptionsByPlanId(
+    planId: string,
+    timeSpan: TimeSpanInS,
+    limit: number = 50
```

Comment:
I set the `limit` for `findActiveSubscriptionsByPlanId` to `50` for these reasons. I'm not sure if I should be concerned about the size of the returned `data` with an expanded `customer`.

Reply:
You shouldn't be concerned. 🙂